websites/E/Emberspark Games/presence.ts

```diff
@@ -0,0 +1,159 @@
+const presence = new Presence({ clientId: "1196469492310618142" }),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+const enum Assets {
+	Icon = "https://i.imgur.com/GVRJihz.png",
+	DesolationIcon = "https://i.imgur.com/TUZ9rwv.png",
+	ConsumerCatalog = "https://i.imgur.com/q3m7DhW.png",
```

Comment:
Assets should be 512x512px. If needed, use online tools like waifu2x to upscale images.

Reply:
they have to be exacly 512x512 or can be bigger?

Reply:
They can be bigger, although according to current guidelines, they should technically be exactly 512x512px. When merging, they will be resized down to 512x512.